frontend/src/pages/Login.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from "react";
+import { Formik, Form } from "formik";
+import { useTranslation } from 'react-i18next';
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+import * as Yup from 'yup';
+
+import Button from "../components/Button";
+import InputField from "../components/InputField";
+
+const mock = new MockAdapter(axios)
+
+mock.onPost("http://localhost:3000/api", { email: "teszt@teszt.com", password: "1234567" })
+    .reply(200, "84848fhgvripuerh98r4gu9hg4ru9hrv")
+
+const Login = () => {
+    const { t, i18n } = useTranslation();
+    const [token, setToken] = useState("");
+
+    useEffect(() => {
+        i18n.changeLanguage("hu")
+    }, [i18n])
+
+    const SignUpSchema = Yup.object().shape({
+        email: Yup.string()
+            .email(t("login.invalidemail"))
+            .required(t("login.noEmail")),
+        password: Yup.string().min(6, t("login.invalidPassword"))
+    })
+
+    const handleResponse = response => {
+        setToken(response.data)
+        console.log(token)
```

Comment:
Console.log-okat szedjük ki, ha már nem kell.

Reply:
Rendben. A CLI warningja miatt hagytam bent. Enélkük nem tudtam beadni a pr-t. Ezzel kapcsolatban lenni is majd kérdésem. A visszakapott token-t gondolom, globálban kellene tárolni. Redux?